demo/blocks_xray/run.py

```diff
@@ -1,14 +1,13 @@
 import gradio as gr
-import random
 import time
 
+disease_values = [0.25, 0.5, 0.75]
+
 def xray_model(diseases, img):
-    time.sleep(4)
```

Comment:
We don't want sleeps in anything we're using to test because it just slows them down for no reason.

Reply:
Agreed